web-no-modal-sdk/blockchain-connection-examples/evm-no-modal-example/src/App.tsx

```diff
@@ -290,6 +290,11 @@ function App() {
   //     uiConsole("provider not initialized yet");
   //     return;
   //   }
+  //
```

Comment:
Why are these comments here?

Is this PR just for evm modal?

Reply:
1. I didn't want to modify the original version because the plugin was already commented.
2. This is a PR for EVM modal and non-modal examples. These are the only ones that have implemented the service wallet.